babel/messages/catalog.py

```diff
@@ -769,6 +769,7 @@ def update(
         no_fuzzy_matching: bool = False,
         update_header_comment: bool = False,
         keep_user_comments: bool = True,
+        ignore_pot_creation_date: bool = False,
```

Comment:
I think

```suggestion
        update_creation_date: bool = True,
```

would be more intuitive? 

The command line parameter could be left `--ignore-pot-creation-date` though.